packages/gatsby/src/utils/webpack-utils.js

```diff
@@ -444,7 +445,7 @@ module.exports = async ({
   plugins.minifyJs = ({ terserOptions, ...options } = {}) =>
     new TerserPlugin({
       cache: true,
-      parallel: true,
+      parallel: !isWsl,
```

Comment:
Just adding some context

```suggestion
      // can't use parallel on WSL because of issue in terser
      // see https://github.com/webpack-contrib/terser-webpack-plugin/issues/21
      parallel: !isWsl,
```

Reply:
Good suggestion. I just [added it here](https://github.com/gatsbyjs/gatsby/pull/12636/commits/b43746ed6d7308891a869277f76d7e4e64e2fa69)